src/main/java/io/vertx/core/http/HttpClientOptions.java

```diff
@@ -1036,6 +1098,8 @@ public int hashCode() {
     result = 31 * result + (sendUnmaskedFrames ? 1 : 0);
```

Comment:
Objects.hashCode(...) ?

Reply:
actually I think this code should be generated like we do generate code for toJson/fromJson which would alleviate us from writing hashcode/equals